src/charm.py

```diff
@@ -211,13 +278,53 @@ def configure_workload_container(self, event) -> None:
         self.unit.status = ActiveStatus(msg)
 
     def _generate_keys_zone(self, name):
-        """Generate hashed name to be used by Nginx's key zone."""
+        """Generate hashed name to be used by Nginx's key zone.
+
+        Args:
+            name: Site name to be encoded.
+
+        Returns:
+            A hashed name to be used by Nginx's key zone.
+        """
         hashed_value = hashlib.md5(name.encode("UTF-8"), usedforsecurity=False)
         hashed_name = hashed_value.hexdigest()[0:12]
         return f"{hashed_name}-cache"
 
+    def _get_nginx_prometheus_exporter_pebble_config(self):
+        """Generate pebble config for the nginx-prometheus-exporter container.
+
+        Returns:
+            Pebble layer config for the nginx-prometheus-exporter container.
+        """
+        return {
+            "summary": "Nginx prometheus exporter",
+            "description": "Prometheus exporter for nginx",
+            "services": {
+                "nginx-exporter": {
+                    "override": "replace",
+                    "summary": "Nginx Exporter",
```

Comment:
Can we include Prometheus in here as well please?

Reply:
IIRC I shouldn't change the name of the service, but can change the summary :+1: